packages/backend/src/core/EmailService.ts

```diff
@@ -156,7 +156,7 @@ export class EmailService {
 	@bindThis
 	public async validateEmailForAccount(emailAddress: string): Promise<{
 		available: boolean;
-		reason: null | 'used' | 'format' | 'disposable' | 'mx' | 'smtp' | 'banned' | 'network' | 'blacklist';
+		reason: null | string;
```

Comment:
I think this shouldn't be to changed 'just stirng', if there is another 'reason' value variants, please append it to here.